src/index.ts

```diff
@@ -3,17 +3,21 @@ import logger from './misc/logger';
 import appRouter from './routes/index';
 import dotenv from 'dotenv';
 import cors from 'cors';
+
+import bodyParser from 'body-parser';
 dotenv.config();
 const app: Application = express();
 app.use(express.json());
 app.use(cors());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
```

Comment:
You wouldn't need body parser. Express already has a method express.json(). [https://stackoverflow.com/questions/47232187/express-json-vs-bodyparser-json](url)

Reply:
ok.